spring-cloud-stream-binder-kafka-core/src/main/java/org/springframework/cloud/stream/binder/kafka/provisioning/KafkaTopicProvisioner.java

```diff
@@ -282,7 +284,7 @@ private void createTopic(AdminClient adminClient, String name, int partitionCoun
 				throw (Error) throwable;
 			}
 			else {
-				throw new ProvisioningException("provisioning exception", throwable);
+				throw new ProvisioningException("Provisioning exception", throwable);
```

Comment:
This message doesn't really add any value, but I see there's no CTOR that just takes a cause, so I guess it's ok; we should consider adding such a CTOR in future.

Also, we should add a TODO - `catch (Throwable)` is bad practice (even though `Error` is re-thrown.